releasenotes/notes/fix-pylibmc-none-span-bug-473849fca820162d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    pylibmc: Fixes an issue where here any use of get/gets/get_multi would break if the tracer was disabled
```

Comment:
```suggestion
    pylibmc: Fixes an issue where using ``ddtrace-run`` or ``ddtrace.patch_all()`` with ``DD_TRACE_ENABLED=False`` would break with get, gets, and get_multi operations on pylibmc Clients.
```